### ISSUE TITLE
Q: text embedding injection

### ISSUE THREAD

**Issue by kgonia** [cloneofsimo/promptplusplus#1]
Hi! Great work!

pls correct me if I'm wrong but you only implemented Textual Inversion part? I'm curious if you also worked on extended conditioning?

![p_pp_sbs 1](https://user-images.githubusercontent.com/4281421/227775958-4ece3dac-28c0-42b8-97d5-af51e9de9c16.jpeg)

**Comment by cloneofsimo** [comment]
Yup this repo (+the paper) is all about extended conditioning 😂

**Comment by kgonia** [comment]
So can I use your code just for providing different prompts to each layer without training Textual Inversion?

**Comment by cloneofsimo** [comment]
Yes. you might want to have a look at how expand + embed works here https://github.com/cloneofsimo/promptplusplus/blob/4efaf4e06f5e3a2c10e3e439c35f53b7ccc56b6e/ppp/utils.py#L80

**Comment by jordandavis16** [comment]
Can this be used with an inpainting pipeline?
